### PR TITLE
Add support for Python 3.6.

### PR DESCRIPTION
**Pull request by mbaechtold**
Closes https://github.com/FactoryBoy/factory_boy/issues/358

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,15 +15,17 @@ matrix:
...
+    - python: "3.6"
+      env: TOXENV=py36-django110-alchemy10-mongoengine010
 
     # Pypy
     - python: "pypy"
       env: TOXENV=py27-django110-alchemy10-mongoengine010
 
     # Linting
-    - python: "3.5"
+    - python: "3.6"
```
Comment (mbaechtold) [inline .travis.yml]: @rbarrois, I'm not sure if these just need to be replaced or rather duplicated for 3.6?

Comment (rbarrois) [inline .travis.yml]: I'd go for replacing: we don't need to run linting/examples on all Python versions ;)

File: tox.ini [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{27,34,35}-django{17,18,19,110}-alchemy10-mongoengine010
+    py{27,34,36}-django{17,18,19,110}-alchemy10-mongoengine010
```
Comment (rbarrois) [inline tox.ini]: Beware: this is removing support for the py35 builds in the tox matrix (which is much wider than what we use in travis).

Comment (mbaechtold) [inline tox.ini]: Nice catch. I fixed that.

**Comment by rbarrois** [comment]
@mbaechtold almost good, I think there is also a reference to the supported Python range in the README.rst file.

Once you add yourself to the ``CREDITS`` file, this will be perfect for merging :smiley:

**Comment by mbaechtold** [comment]
@rbarrois, I've done that. Thank you :-)

**Comment by rbarrois** [comment]
@mbaechtold Awesome, thanks!

Merging now :smile:

**Pull request merged**